pkg/log/tracelog.go

```diff
@@ -0,0 +1,72 @@
+package log
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/uptrace/opentelemetry-go-extra/otelzap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	_traceLogger = otelzap.New(zap.NewNop())
+)
+
+// T is a wrapper of otelzap.Ctx, returns a logger with context.
+func T(ctx context.Context) otelzap.LoggerWithCtx {
```

Comment:
listen logs from `L()`

Reply:
When `initTraceLogger`, T(ctx) and L() use the same zap logger. 

In different , T(ctx) will additionally attach some trace fields and send them to the trace provider(jaeger/signoz)

Reply:
ok. Does the tracer work as before with this pr?